pages/faq/index.public.js

```diff
@@ -56,9 +56,9 @@ Atualmente, o único espaço de anúncio disponível é o de [publicações patr
       question: 'Como funciona uma publicação patrocinada?',
       answer: `_Esse tipo de anúncio está em desenvolvimento, então está em constante evolução. Você pode acompanhar o que está sendo feito no [issue #1491 do GitHub](https://github.com/filipedeschamps/tabnews.com.br/issues/1491)._
 
-No topo das listas de conteúdos [Relevantes](/) e [Recentes](/recentes/pagina/1), e também nas páginas de publicações e comentários, após o conteúdo principal, uma publicação patrocinada escolhida de forma aleatória é exibida. Caso a publicação tenha um link de "**fonte**", o visitante que clicar no título da publicação será redirecionado para o link. Caso o link seja para um site externo, o domínio será identificado após o título, por exemplo: \`Título da publicação patrocinada (site-externo.com.br)\`.
+No topo das listas de conteúdos [Relevantes](/) e [Recentes](/recentes/pagina/1), e também nas páginas de publicações e comentários, após o conteúdo principal, uma publicação patrocinada escolhida de forma aleatória é exibida como um _banner_. Caso a publicação tenha um link de "**fonte**", o visitante que clicar no título da publicação será redirecionado para o link. Caso o link seja para um site externo, o domínio será identificado após o título, por exemplo: \`Título da publicação patrocinada (site-externo.com.br)\`.
 
-Para criar uma publicação patrocinada, você investirá **100 TabCash**. A cada dia que passar, **10 TabCash** serão consumidos da publicação patrocinada.
+Para criar uma publicação patrocinada, você investirá **100 TabCash** no orçamento dela. A cada dia que passar, **10 TabCash** serão consumidos desse orçamento. Quando o orçamento acabar, a publicação patrocinada não aparecerá mais como um banner, mas ela continuará visível na página Recentes (em [Classificados](/recentes/classificados/1) e [Todos](/recentes/todos/1)) e também em Classificados no seu perfil.
```

Comment:
Não apenas pelos questionamentos que tivemos hoje, mas eu já estava querendo fazer a sugestão abaixo. Daí acabo de assistir a [aula de hoje do curso.dev](https://curso.dev/web/git-commit-escopo), e vejo o PR aberto, tudo relacionado... haha... Então lá vai...

Acredito que devemos adicionar aqui no FAQ apenas o que já estiver implementado. Ou então deixar bem claro que é só um plano futuro e que pode mudar. Por isso, eu removeria a maior parte desse trecho.

```suggestion
Para criar uma publicação patrocinada, você investirá **100 TabCash** no orçamento dela. Ainda não está definido como o orçamento será consumido e ainda não é possível alterar o valor do orçamento.
```

---

Ao invés de fazer a atividade proposta na aula de hoje lá dentro da plataforma do curso, vou fazer aqui mesmo. 😅

É para dar a opinião sobre a mudança na documentação entrar no mesmo commit da alteração no código. Eu acredito que depende se a mudança na documentação é obrigatória ou opcional. Se a mudança no código torna a documentação obsoleta, então ela deve obrigatoriamente ser atualizada junto. Já se a mudança no código apenas abre novas possibilidades, sem modificar as anteriores, então acredito que a atualização da documentação é uma etapa seguinte na evolução natural do projeto, então pode vir em um commit posterior.

Reply:
> Daí acabo de assistir a [aula de hoje do curso.dev](https://curso.dev/web/git-commit-escopo), e vejo o PR aberto, tudo relacionado... haha...

Não to conseguindo acreditar até agora o timing disso tudo 😂 😂 😂 🤝